### ISSUE TITLE
Insufficient placeholder contrast

### ISSUE THREAD

**Issue by Kyzyl-ool** [gravity-ui/uikit#1334]
In Select, TextInput and TextArea components there is a lack of contrast. Current is 2.09 when the expected value is 4.5:
<img width="238" alt="Screenshot 2024-02-12 at 14 00 20" src="https://github.com/gravity-ui/uikit/assets/30153778/8e5f9cdc-43a7-4769-bfe1-bf25734221de">
https://preview.gravity-ui.com/uikit/?path=/story/components-inputs-select--showcase
```
Element has insufficient color contrast of 2.09 (foreground color: #b3b3b3, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1
```
Can we use a color with higher contrast value for placeholders?

**Comment by ykamendrovskiy** [comment]
We'll review all the subtle color values when updating the default theme
@amje

**Comment by amje** [comment]
Will be revisited on theme update
